ansible/deploy_mailrelay.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Install postfix"
+  hosts: all
+  sudo: yes
+  tasks:
+    - package: name=postfix state=present
+
+- name: "mail relay deploy"
+  hosts: mailrelay
+  sudo: yes
+  tasks:
+    - name: "sets our relayhost"
+      lineinfile:
+        dest: /etc/postfix/main.cf
+        regexp: '^relayhost ='
+        line: "relayhost = {{ mailrelay }}"
```

Comment:
maybe "mailrelay" is ok for the inventory, since all of those are hosts, but maybe use smtp_server instead of the "mailrelay" variable. would that do the trick?